### ISSUE TITLE
tag input autocomplete Items drop-down appears in odd position after entering maximumItems.

### ISSUE THREAD

**Issue by pycraft114** [Gbuomprisco/ngx-chips#603]
PLEASE MAKE SURE THAT:
- you searched similar issues online (9/10 issues in this repo are solved by googling, so please...do it) 
- you provide an online demo I can see without having to replicate your environment
- you help me by debugging your issue, and if you can't, do go on filling out this form

**I'm submitting a ...**  (check one with "x")
```
[ v ] bug report => search github for a similar issue or PR before submitting
[  ] support request/question

Notice: feature requests will be ignored, submit a PR if you'd like
```

**Current behavior**

tag input drop-down that shows autoCompleteItems appears in odd position after entering all maximum items.
<img width="1440" alt="2017-11-06 5 50 24" src="https://user-images.githubusercontent.com/24728657/32432522-4ec6f00a-c31b-11e7-836b-008b7eb54b62.png">
picture above shows that drop down appears at expected position when trying to enter the last item.
(the maxItem property is set to 3)

<img width="1440" alt="2017-11-06 5 51 02" src="https://user-images.githubusercontent.com/24728657/32432640-a5d13e28-c31b-11e7-96ae-35373b3729fd.png">
And picture above shows that the drop down appears at odd position (upper left corner) when entering all available item is finished.

**Expected behavior**

when max number of items is entered, the drop down should not be displayed. 

**Minimal reproduction of the problem with instructions (if applicable)**

**What do you use to build your app?. Please specify the version**

``
"dependencies": {
    "@angular/animations": "4.4.6",
    "@angular/cdk": "2.0.0-beta.12",
    "@angular/common": "4.4.6",
    "@angular/compiler": "4.4.6",
    "@angular/compiler-cli": "4.4.6",
    "@angular/core": "4.4.6",
    "@angular/forms": "4.4.6",
    "@angular/http": "4.4.6",
    "@angular/language-service": "4.4.6",
    "@angular/material": "2.0.0-beta.12",
    "@angular/platform-browser": "4.4.6",
    "@angular/platform-browser-dynamic": "4.4.6",
    "@angular/platform-server": "4.4.6",
    "@angular/router": "4.4.6",
    "@types/jasmine": "2.6.0",
    "@types/jasminewd2": "2.0.3",
    "ag-grid": "13.2.0",
    "ag-grid-angular": "13.2.0",
    "angularfire2": "4.0.0-rc.2",
    "apollo-angular": "0.13.1",
    "apollo-client": "1.9.2",
    "bootstrap": "4.0.0-beta",
    "codelyzer": "3.2.0",
    "core-js": "2.4.1",
    "firebase": "4.3.1",
    "graphql-tag": "2.4.2",
    "hammerjs": "2.0.8",
    "jasmine-core": "2.8.0",
    "jasmine-spec-reporter": "4.2.1",
    "jquery": "3.2.1",
    "karma": "1.7.1",
    "karma-coverage-istanbul-reporter": "1.3.0",
    "ngx-chips": "1.5.3",
    "popper.js": "1.12.5",
    "rxjs": "5.4.2",
    "tether": "1.4.0",
    "ts-node": "3.3.0",
    "zone.js": "0.8.14"
  },
  "devDependencies": {
    "@angular/cli": "1.4.9",
    "@angular/compiler-cli": "4.4.6",
    "@angular/language-service": "4.4.6",
    "@types/jasmine": "2.5.53",
    "@types/jasminewd2": "2.0.2",
    "@types/node": "6.0.88",
    "apollo-codegen": "0.17.0",
    "codelyzer": "3.1.1",
    "jasmine-core": "2.6.2",
    "jasmine-spec-reporter": "4.1.0",
    "karma": "1.7.0",
    "karma-chrome-launcher": "2.1.1",
    "karma-cli": "1.0.1",
    "karma-coverage-istanbul-reporter": "1.2.1",
    "karma-jasmine": "1.1.0",
    "karma-jasmine-html-reporter": "0.2.2",
    "protractor": "5.1.2",
    "rollup": "0.50.0",
    "rollup-plugin-commonjs": "8.2.1",
    "rollup-plugin-node-resolve": "3.0.0",
    "rollup-plugin-uglify": "2.0.1",
    "ts-node": "3.2.0",
    "tslint": "5.3.2",
    "typescript": "2.5.3"
  }
``

**Angular version:** 

4.4.6

**ngx-chips version:**

1.5.3

**Browser: Chrome

**Comment by Gbuomprisco** [comment]
Hi, are you using appendToBody=false?

**Comment by pycraft114** [comment]
@Gbuomprisco Yes appendToBody property is set to false due to necessary reason. In this case, how should i fix this issue?

**Comment by Gbuomprisco** [comment]
what's the output if you set it to true? I don't see any solution in this case

**Comment by pycraft114** [comment]
@Gbuomprisco the same bug happens even tho when it is set to true. What does the drop down depends on ? whether to show or not

**Comment by Gbuomprisco** [comment]
If you search in the threads you will find a workaround for that issue, you'll need to set that to true and edit some style

**Comment by pycraft114** [comment]
@Gbuomprisco are there any possible way to fix this while appendToBody property is still set to false?

**Comment by pycraft114** [comment]
@Gbuomprisco 
```
If you search in the threads you will find a workaround for that issue, you'll need to set that to true and edit some style
``` true meaning by appendToBody?

**Comment by Gbuomprisco** [comment]
Yep

**Comment by lud42** [comment]
Hello i've got the same problem, i set "appendToBody" to true but the dropdown always appears in odd position

**Comment by lud42** [comment]
Hello i've got the same problem, i set "appendToBody" to true but the dropdown always appears in odd position @Gbuomprisco

**Comment by pycraft114** [comment]
@Gbuomprisco Made a pull request for this. 
https://github.com/Gbuomprisco/ngx-chips/pull/640
Hide autoComplete option dropdown when amount of items are reached to ```maxItems```

**Comment by pycraft114** [comment]
@lud42 this issue is solved at version 1.6.2 closing this issue
